cbmc/proofs/Makefile.common

```diff
@@ -42,9 +42,16 @@ CFLAGS += $(CFLAGS2) $(INC) $(DEF) -std=gnu99
 %.goto : %.c
 	$(GOTO_CC) -o $@ $(CFLAGS) $<
 
+# Use the generic implementations of atomic operations.
+# Using the gcc atomic intrinsics causes coverage computation to fail.
+# Define the generic flag here, and replace *_gcc.h with *_generic.h below.
+DEF += -DIOT_ATOMIC_GENERIC=1
+
 $(MQTT)/build:
 	(mkdir -p $(MQTT)/build; cd $(MQTT)/build; cmake ..) \
 	       > $(ENTRY)0.txt 2>&1
+	cp $(MQTT)/ports/common/include/atomic/iot_atomic_generic.h \
```

Comment:
Delete this generic stuff, for the reason above.

Reply:
Done!